src/models/desk.ts

```diff
@@ -41,7 +41,7 @@ export default class Desk extends Collider {
       t.enqueue(
         this,
         deskTextureName,
-        (renderer: PIXI.Renderer | PIXI.AbstractRenderer): PIXI.Texture =>
+        (renderer: PIXI.Renderer): PIXI.Texture =>
```

Comment:
`PIXI.AbstractRenderer` is no longer a thing in PixiJS 7, so this is removed across various Pixi models and the texture generator.